translations/monero-core_fr.ts

```diff
@@ -177,381 +144,389 @@
         <source>Prove/check</source>
         <translation>Prouver/Vérifier</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="440"/>
+        <location filename="../LeftPanel.qml" line="458"/>
         <source>K</source>
         <translation>K</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="369"/>
+        <location filename="../LeftPanel.qml" line="387"/>
         <source>History</source>
         <translation>Historique</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="96"/>
+        <location filename="../LeftPanel.qml" line="137"/>
         <source>View Only</source>
         <translation>Audit</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="108"/>
+        <location filename="../LeftPanel.qml" line="124"/>
         <source>Testnet</source>
         <translation>Testnet</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="320"/>
+        <location filename="../LeftPanel.qml" line="124"/>
+        <source>Stagenet</source>
+        <translation>Stagenet</translation>
+    </message>
+    <message>
+        <location filename="../LeftPanel.qml" line="338"/>
         <source>Address book</source>
         <translation>Carnet d&apos;adresses</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="321"/>
+        <location filename="../LeftPanel.qml" line="339"/>
         <source>B</source>
         <translation>B</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="370"/>
+        <location filename="../LeftPanel.qml" line="388"/>
         <source>H</source>
         <translation>H</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="392"/>
+        <location filename="../LeftPanel.qml" line="410"/>
         <source>Advanced</source>
         <translation>Avancé</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="393"/>
+        <location filename="../LeftPanel.qml" line="411"/>
         <source>D</source>
         <translation>D</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="415"/>
+        <location filename="../LeftPanel.qml" line="433"/>
         <source>Mining</source>
         <translation>Mine</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="416"/>
+        <location filename="../LeftPanel.qml" line="434"/>
         <source>M</source>
         <translation>M</translation>
     </message>
     <message>
-        <location filename="../LeftPanel.qml" line="508"/>
+        <location filename="../LeftPanel.qml" line="480"/>
+        <source>Shared RingDB</source>
+        <translation>Shared RingDB</translation>
```

Comment:
or "BDD de Cercle partagée" ?

Reply:
more "Base de données partagée de cercles"

Reply:
"ring" -> "anneau" plutôt que "cercle", non?

Reply:
It's about ring signature, which is "signature de cercle". See [wikipedia](https://fr.wikipedia.org/wiki/Signature_de_cercle)

Reply:
WP is not an authority, moreover please read the very article you pointed to, the very first line states "aussi appelé signature d’anneau".

The reason why "anneau" seems better to me is: "Circle"->"cercle" and "ring"->"anneau"

Reply:
Ok my bad. But could we make this consistent against all the translations? If it's to be anneaux then I have to update the website

Reply:
I also choose to stay with cercle because of gui "taille de cercle" for ring size

Reply:
The google's result test gives cercle just above anneaux.
- signature de cercle: 624 000 results
- signature d'anneau: 585 000 results.

I do understant your concern, but we have to make it global accross all translations.
Ping @assylias to have another pair of eyes here.

Reply:
Hi there - I don't know to be honest, but the few research papers in French that I found online use "signature de cercle" and "signature de cercle a seuil".

Also a Google search on "signature d'anneau" (with the quotes) returns 2k entries, vs 43k for "signature de cercle".

Finally, "anneau" has a specific meaning in maths, like in "anneau euclidien", which has little to do with ring signatures.

I would personally go for "cercle".

Reply:
As a ring signature is a signature made by someone in a small group of possible signers, it might make more sense to use the word "cercle" (as in "cercle d'amis").